server/config.ts

```diff
@@ -130,7 +130,7 @@ export default {
         response: Number(get('PRISONER_SEARCH_API_TIMEOUT_RESPONSE', 10000)),
         deadline: Number(get('PRISONER_SEARCH_API_TIMEOUT_DEADLINE', 10000)),
       },
-      agent: new AgentConfig(Number(get('PRISONER_SEARCH_API_TIMEOUT_RESPONSE', 10000))),
+      agent: new AgentConfig(Number(get('PRISONER_SEARCH_API_TIMEOUT_RESPONSE', 20000))),
```

Comment:
I think this is for prisoner search API, think you need line 93 instead

Reply:
Thank you!